src/simulator_core/adapter/transforms/mappers.py

```diff
@@ -32,6 +32,27 @@
 from simulator_core.simulation.mappers.mappers import EsdlMapperAbstract
 
 
+def connect_connected_asset(connected_py_assets, junction, py_assets_list,
+                            py_junction_list):
+    """Method to connect assets connected to one asset to the same junction.
+
+    :param connected_py_assets: List of connected assets
+    :param junction: Junction to connect the assets to
+    :param py_assets_list: List of assets
+    :param py_junction_list: List of junctions
+    :return: None
+    """
+    for connected_py_asset in connected_py_assets:
+        index = [py_asset_temp.asset_id for py_asset_temp in py_assets_list].index(
+            connected_py_asset[0]
+        )
+        if connected_py_asset[1] == Port.In:  # from
+            py_assets_list[index].set_from_juction(from_junction=junction)
+        else:  # to
+            py_assets_list[index].set_to_junction(to_junction=junction)
+    py_junction_list.append(junction)
```

Comment:
Gaat dit goed zonder dat je een return hebt naar py_junction_list? Zou het niet netter zijn om gewoon een lijst weg te schrijven:

Willen we daarnaast hier niet type hinting toevoegen? 
```suggestion
def connect_connected_asset(connected_py_assets, junction, py_assets_list,
                            ) -> List[Junction]:
    """Method to connect assets connected to one asset to the same junction.

    :param connected_py_assets: List of connected assets
    :param junction: Junction to connect the assets to
    :param py_assets_list: List of assets
    :return py_junction_list: List of junctions    
    """
    for connected_py_asset in connected_py_assets:
        index = [py_asset_temp.asset_id for py_asset_temp in py_assets_list].index(
            connected_py_asset[0]
        )
        if connected_py_asset[1] == Port.In:  # from
            py_assets_list[index].set_from_juction(from_junction=junction)
        else:  # to
            py_assets_list[index].set_to_junction(to_junction=junction)
    py_junction_list.append(junction)
```

Reply:
Removed the junction list, since it did not make sense to add the junction in this method, it could also be done outside.